api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -0,0 +1,131 @@
+  <style>
+    body {
+      background-color: #F1F1F1;
+      padding-left: 1rem;
+      padding-top: 1rem;
+      width: 30rem;
+    }
+    * {
+      font-family: "Gotham SSm A", "Gotham SSm B", sans-serif;
+    }
+    .welcome-message {
+      margin-top: 2rem;
+      margin-bottom: 1rem;
+      color: #302C71;
+      font-size: 30px;
+      font-weight: 300;
+      line-height: 36px;
+    }
+
+    .text {
+      color: rgba(0,0,0,0.87);
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 22px;
+      margin-bottom: 1rem;
+    }
+
+    .text.user-field {
+      width: 8rem;
+      display: inline-block;
+      margin-bottom: 0;
+    }
+    .user-info {
+      font-weight: bold;
+      display: inline-block;
+    }
+    .card {
+      height: 144px;
+      width: 540px;
+      border-radius: 4px;
+      background-color: #FFFFFF;
+      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      margin-bottom: 1rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+
+    div.card {
+      cursor: default;
+    }
+
+    .bullet {
+      height: 31px;
+      width: 31px;
+      border-radius: 15px;
+      background-color: #302C71;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      color: #FFFFFF;
+    }
+
+    .card-text {
+      width: 194px;
+      color: #302C71;
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 19px;
+    }
+
+    .step-img {
+      height: 120px;
+      width: 250px;
+    }
+    .link {
+      display: inline;
+      color: #2691D0;
+    }
+    .button {
+      height: 52px;
+      width: 178px;
+      border-radius: 5px;
+      background-color: #302C71;
+      font-size: 16px;
+      font-weight: 500;
+      line-height: 19px;
+      color: #FFFFFF;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+  </style>
+</head>
+<body>
+  <img src="assets/logo-registration-non-signed-in.svg">
+  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+  <div><div class="text user-field">Your User Name: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
+  <div><div class="text user-field">Your Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <br>
+  <div class="text">Quick guide on how to get started:</div>
+  <a class="card" href="https://accounts.google.com/signin/oauth/oauthchooseaccount?hd={{TARGET-DOMAIN}}&client_id={{CLIENT-ID}}.apps.googleusercontent.com&as=1ivZqDJMGPEMvWasjAquSQ&destination={{URL-INPUT}}&approval_state=!ChRqd2tjYnk1b040d3VUMzNpZXBpQRIfczlfc0R4d2dERHdUQU9FaERQUmpBazFLYU13VFBSWQ%E2%88%99AB8iHBUAAAAAWxgpw3y6ICN3aJ_etNsSGfEmZbFNYV2d&xsrfsig=AHgIfE85mPI6c5DfCze82tpt7pbw8-XBRg&flowName=GeneralOAuthFlow">
```

Comment:
Where is this link actually supposed to bring the user after they login? Seems like we may want them to go to 2SV setup rather than workbench.

Also, unless you did something special with this, I'd guess these approval_state or xsrfsig params are completely specific to the time when you generated this link and should not be included here. I would recommend making this URL as minimal as possible to avoid depending on Google signin internals.

Reply:
I am hoping to go somewhere where we restrict the domain. Presumably that is possible with linking to the 2SV setup, via a query param? Is it possible to link to that?

Reply:
Or can I just set the destination URL and leave that?

Reply:
I haven't looked deeply at this, but my guess would be that you could restrict the target domain via "hd", then set the destination to the 2sv URL. I would test that though.

Reply:
Even better, if it's possible to hint login for a specific user, i.e. the one you just created.

Reply:
Yeah. Possibly. 

destination doesn't actually seem to work, which is strange...

Reply:
Have tested hd, and it doesn't look like I can restrict to exact email.